lib/rihanna/job.ex

```diff
@@ -59,6 +60,14 @@ defmodule Rihanna.Job do
   end
   ```
 
+  You can define a `priority/0` function which will be called if no priority
+  is set when a job is enqueued. It should return a single integer.
+  If you don't define this callback it will default to the lowest priority, 20.
```

Comment:
Is it 20 or 50? And why these number choices?

Reply:
is 10 higher or lower than 20?

Reply:
10 is higher priority. 1 is the highest priority. I will make this consistent and add some docs explaining.